### PR TITLE
resolve #3 (download tarballs rather than wheels)

### LINKED ISSUE #3: pip3 issues
* pip 9.x downloads wheels instead of tar.gz of pure python packages, which can be a problem if the wrong python is being used. There's a platform flag you can give to pip 9
* Need to disable pip's caching, we don't need it for this

> Migrated from https://github.com/robotpy/robotpy-wpilib/issues/249

### PR DESCRIPTION
**Pull request by czhao39**

### CODE REVIEW

File: robotpy_installer/installer.py [hunk 1]
```diff
@@ -933,6 +934,8 @@ def download_pip(self, options):
         
         # Use pip install --download to put packages into the cache
         pip_args = ['install',
+                    '--cache-dir',
```
Comment (virtuald) [inline robotpy_installer/installer.py]: Why not use `--no-cache`?

**Comment by virtuald** [comment]
Also, I'm not opposed to wheels -- the main problem is just that the installer doesn't copy them over (I think it just looks for tar.gz?). I think maybe we could force pip >= 9 and then use `--platform` to make sure that if we do download a wheel, it's at least not a binary wheel.

File: robotpy_installer/installer.py (hunk 1 above)
Comment (czhao39) [inline robotpy_installer/installer.py]: My bad, I didn't realize that would work, too.

**Comment by virtuald** [comment]
Also, if you're going to mess with pip, you should add `--disable-pip-version-check`, as that slows things down a tiny bit.

**Comment by czhao39** [comment]
Okay I really over-complicated things. Just adding `--no-cache` does the same as what I did.

**Comment by virtuald** [comment]
You don't have to close the PR, you could have just force-pushed to this branch.

**Pull request closed without merging**